src/main/java/com/volunteer/enterprise/VolunteerService.java

```diff
@@ -2,14 +2,16 @@
 
 import org.springframework.stereotype.Component;
 
-
+import dto.Volunteer;
 import java.util.List;
 
 @Component
 public interface VolunteerService {
+    // Should pass dto objects in this methods
+    void save(Volunteer volunteer);
 
-    void save(VolunteerService volunteerService);
-
+    // Your controller called the delete method but there was not one found
+    void delete(Integer i);
 
```

Comment:
Discussion/commentary does not belong in source code.  It's clutter.  It belongs in the GitHub pull request.